cas/ut/client_ut.c

```diff
@@ -202,14 +199,15 @@ static void casc_ut_init(struct m0_rpc_server_ctx *sctx,
 			 struct cl_ctx            *cctx)
 {
 	int rc;
-
+	sctx->rsx_xprts = m0_net_all_xprt_get();
+	sctx->rsx_xprts_nr = m0_net_xprt_nr();
```

Comment:
These can be at old place, do you see any issue ?

Reply:
No.
In original code, the "cs_xprts" was defined as a static global variable. It can be initialized from const.

But, now, these functions, "m0_net_all_xprt_get()" & "m0_net_xprt_nr()" can only be called when the transports are registered in initialization code.